src/sidebar/components/test/annotation-test.js

```diff
@@ -917,9 +915,39 @@ describe('annotation', function() {
     });
 
     describe('#shouldShowLicense', function() {
-      unroll(
-        'returns #expected if #case_',
-        function(testCase) {
+      [
+        {
+          case_: 'the annotation is not being edited',
+          draft: null,
+          group: groupFixtures.open,
+          expected: false,
+        },
+        {
+          case_: 'the draft is private',
+          draft: draftFixtures.private,
+          group: groupFixtures.open,
+          expected: false,
+        },
+        {
+          case_: 'the group is private',
+          draft: draftFixtures.shared,
+          group: groupFixtures.private,
+          expected: false,
+        },
+        {
+          case_: 'the draft is shared and the group is open',
+          draft: draftFixtures.shared,
+          group: groupFixtures.open,
+          expected: true,
+        },
+        {
+          case_: 'the draft is shared and the group is restricted',
+          draft: draftFixtures.shared,
+          group: groupFixtures.restricted,
+          expected: true,
+        },
+      ].forEach(testCase => {
+        it(`returns #expected if ${testCase.case_}`, () => {
```

Comment:
`#expected` here needs to be replaced with `${expected}`

Reply:
good catch!